app/models/user.py

```diff
@@ -78,6 +78,7 @@ class User(SoftDeletionModel):
     was_registered_with_order = db.Column(db.Boolean, default=False)
     last_accessed_at = db.Column(db.DateTime(timezone=True))
     created_at = db.Column(db.DateTime(timezone=True), default=datetime.now(pytz.utc))
+    deleted_at = db.Column(db.DateTime(timezone=True), default=None)
```

Comment:
user model class is a subclass of `SoftDeletionModel` by virtue of which it already has a `deleted_at` column. In fact that's where the `deleted_at` timestamp is stored. What are you trying to do here? Making changes to model files actually adds a column in the database. There already is a `deleted_at` column for users.

Reply:
@CosmicCoder96 Actually, I was just trying to  change the query so that the `deleted_at=None` can be executed in the filter. I was aware of the presence of `deleted_at` element but it was not working as expected (giving a `401 Authentication Error`)

So i added a deleted_at field manually to see if it works then and it did. 
I think I will retest and see if it was some error from my side

Reply:
@CosmicCoder96 My bad, it was incorrect testing from my side. Please review now